### DISCUSSION TITLE
Inventory item count overlay during skilling

### DISCUSSION THREAD

**Discussion by Speeddymon** [runelite/runelite#17106] [Ideas]
It'd be nice to have a small overlay which displays the number of free spaces remaining in your inventory when skilling things like fishing. This way I don't have to keep my inventory open all the time to optimally track when to head back to a range, the GE, or the bank.

**Comment by Alexsuperfly** [comment]
https://runelite.net/plugin-hub/show/inventory-summary

**Reply by Speeddymon** [reply]
I was looking for something smaller but hadn't thought to check plugins, as I am new to runelite. I found one that does what I was looking for, thanks.
